examples/waspleau/Notes.md

```diff
@@ -0,0 +1,198 @@
+2021-12-16T02:18:54.208403+00:00 app[web.1]:
+2021-12-16T02:18:54.208404+00:00 app[web.1]: Migration name: 20211208191826_dummy
+2021-12-16T02:18:54.208404+00:00 app[web.1]:
+2021-12-16T02:18:54.208404+00:00 app[web.1]: Database error code: 42601
+2021-12-16T02:18:54.208405+00:00 app[web.1]:
+2021-12-16T02:18:54.208405+00:00 app[web.1]: Database error:
+2021-12-16T02:18:54.208406+00:00 app[web.1]: db error: ERROR: syntax error at or near "AUTOINCREMENT"
+2021-12-16T02:18:54.208406+00:00 app[web.1]:
+2021-12-16T02:18:54.208406+00:00 app[web.1]:
+2021-12-16T02:18:54.220572+00:00 app[web.1]: npm ERR! code ELIFECYCLE
+2021-12-16T02:18:54.220779+00:00 app[web.1]: npm ERR! errno 1
+2021-12-16T02:18:54.224811+00:00 app[web.1]: npm ERR! server@0.0.0 db-migrate-prod: `prisma migrate deploy --schema=../db/schema.prisma`
+2021-12-16T02:18:54.224850+00:00 app[web.1]: npm ERR! Exit status 1
+2021-12-16T02:18:54.224925+00:00 app[web.1]: npm ERR!
+2021-12-16T02:18:54.224925+00:00 app[web.1]: npm ERR! Failed at the server@0.0.0 db-migrate-prod script.
+2021-12-16T02:18:54.224953+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
+2021-12-16T02:18:54.229064+00:00 app[web.1]:
+2021-12-16T02:18:54.229202+00:00 app[web.1]: npm ERR! A complete log of this run can be found in:
+2021-12-16T02:18:54.229274+00:00 app[web.1]: npm ERR!     /app/server/.npm/_logs/2021-12-16T02_18_54_225Z-debug.log
+2021-12-16T02:18:54.250005+00:00 app[web.1]: npm ERR! code ELIFECYCLE
+2021-12-16T02:18:54.250216+00:00 app[web.1]: npm ERR! errno 1
+2021-12-16T02:18:54.254548+00:00 app[web.1]: npm ERR! server@0.0.0 start-production: `npm run db-migrate-prod && NODE_ENV=production node ./src/server.js "npm" "run" "start-production"`
+2021-12-16T02:18:54.254618+00:00 app[web.1]: npm ERR! Exit status 1
+2021-12-16T02:18:54.254695+00:00 app[web.1]: npm ERR!
+2021-12-16T02:18:54.254741+00:00 app[web.1]: npm ERR! Failed at the server@0.0.0 start-production script.
+2021-12-16T02:18:54.254783+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
+2021-12-16T02:18:54.258329+00:00 app[web.1]:
+2021-12-16T02:18:54.258473+00:00 app[web.1]: npm ERR! A complete log of this run can be found in:
+2021-12-16T02:18:54.258545+00:00 app[web.1]: npm ERR!     /app/server/.npm/_logs/2021-12-16T02_18_54_255Z-debug.log
+2021-12-16T02:18:54.433588+00:00 heroku[web.1]: Process exited with status 1
+2021-12-16T02:18:54.568000+00:00 heroku[web.1]: State changed from starting to crashed
+2021-12-16T02:19:01.210559+00:00 app[api]: Deployed web (f5ce2d583e48) by user thelastemail@gmail.com
+```
+
+Had to delete and re-add database. Probably when I originally deployed with sqlite there it got into a funky state.
```

Comment:
Oh wow hm!

Reply:
Yeah that part was super strange 🤔

Reply:
is this easy to repro? Would be good to create an issue if yes

Reply:
Yep, I can try to repro it sometime and will create an issue if so. I will add a TODO to remind me in case I cannot do before merge.